lib/modules/datasource/github-releases/cache/cache-base.ts

```diff
@@ -365,4 +377,27 @@ export abstract class AbstractGithubDatasourceCache<
 
     return result;
   }
+
+  public isChangelogReleaseFresh(
```

Comment:
```suggestion
  public isVeryRecentRelease(
```

Would this be more understandable?

Reply:
Not necessarily, because very recent release can also be 1 month ago

Reply:
`newChangelogDetected` or `newChangelogReleaseDetected` maybe?